### DISCUSSION TITLE
Installation from forked source installs old version - question

### DISCUSSION THREAD

**Discussion by bigtedde** [borgbackup/borg#7587] [Q&A] [answered]
When I install Borg following the [instructions](https://borgbackup.readthedocs.io/en/master/installation.html#using-git) from source  `git@github.com:borgbackup/borg.git` it installs `borgbackup-2.0.0b6.dev167+g852172f5`

However, when I follow the same instructions to install from my forked source `git@github.com:bigtedde/borg.git` it installs `borgbackup-0.1.dev7874+g852172f`

I deleted my fork and created a new one, to the same effect. The older borg version causes fatal issues when running the testing suite. Is there a way to install the new version from the forked source?

@ThomasWaldmann @m3nu

**Comment by m3nu** [comment]
Look into the available branches and tags. You may not use the one you want.

`git log` also helps to see the list of commits and their date.

**Reply by bigtedde** [reply]
I am in master for both, and both are on the same commit. However, when I try to install from my forked source it always installs `borgbackup-0.1.dev7874+g852172f` even tho this code is identical. What do you think would cause this discrepancy?

**Reply by m3nu** [reply, marked as answer]
You need to see how this version string is built. In this case it looks like the Git tag, so you may not have pulled all tags.

**Reply by RonnyPfannschmidt** [reply]
borg uses setuptools_scm, which looks at the git tags, if your fork doesnt have recent ones, just fech them from upstream and push the tags - afterwards the ersion vschould work

**Reply by bigtedde** [reply]
@m3nu @RonnyPfannschmidt Thank you both for your help! I pulled the tags and my forked version is installing as 2.0.x now 👍
